learn/cloud/monitoring.mdx

```diff
@@ -0,0 +1,67 @@
+description: Meilisearch Cloud offers in-depth search analytics to help monitor your application performance.
+sidebarDepth: 3
+---
+
+# Monitoring
+
+Enable Meilisearch Cloud monitoring to keep track of application performance and service status. 
+
+## Requirements
+
+You must have a [Meilisearch Cloud](https://meilisearch.com/cloud?utm_campaign=oss&utm_source=docs&utm_medium=monitoring) account to access monitoring metrics.
+
+## Activating monitoring
+
+### Enable monitoring in the project overview
+
+Log into your Meilisearch Cloud account and navigate to your project's overview. Find the "Analytics and monitoring" section and click on the "Enable analytics monitoring" button:
+
+![The analytics section of the project overview. It shows one button, "Enable analytics and monitoring", and a short explanation of both features.](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/cloud-analytics/01-analytics-enable.png)
+
+Meilisearch Cloud will begin processing your request. The "Analytics and monitoring" section will update when the feature is enabled.
+
+<Capsule intent="note" title="Monitoring and analytics">
+Activating monitoring will automatically activate [analytics](/learn/cloud/analytics). 
+</Capsule>
+
+### Update URLs in your application
+
+When you enable monitoring, Meilisearch Cloud changes your project's API URL. Meilisearch Cloud is only able to track metrics for queries sent to this URL. 
+
+Update your application so all API requests point to the new URL:
+
+```sh
+curl \
+  -X POST 'http://edge.meilisearch.com/indexes/products/search' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{ "q": "green socks" }'
+```
+
+**The previous API URL will remain functional**, but requests targeting it will not send any data to the analytics interface.
+
+## Deactivating monitoring
+
+### Disable monitoring in the project overview
+
+Log into your Meilisearch Cloud account and navigate to your project's overview. Find the "Analytics and monitoring" section and press the "Disable analytics and monitoring" button:
+
+![The analytics section of the project overview. It shows one button, "Disable analytics and monitoring", and a short explanation of both features.](https://raw.githubusercontent.com/meilisearch/documentation/cloud-analytics-guide/assets/images/main/02-analytics-disable.png)
+
+<Capsule intent="note" title="Monitoring and analytics">
+Deactivating monitoring will automatically deactivate [analytics](/learn/cloud/analytics). 
+</Capsule>
+
+### Update URLs in your application
+
+Disabling analytics changes your API URL. Update your application so all API requests point to the correct URL:
+
+```sh
+curl \
+  -X POST 'http://PROJECT_URL/indexes/products/search' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{ "q": "green socks" }'
+```
+
+**The previous API URL will remain functional**, but Meilisearch recommends not using it after disabling analytics in your project.
```

Comment:
is this true both ways? ie. keys created before enabling AND keys creating before disabling?

Reply:
The correct behavior is:

When the user enables the monitoring or the search analytics, it should completely replace their usage with the new `edge.meilisearch.com`.

If, for some reason, the user creates a key without using the edge URL, that new key will not be synced, so it will not work.

So Amelie's statement is true only in this case.

Reply:
ok, so to be sure I understand it correctly:

- if I create a custom API key, then activate monitoring, the API key will work
- if I activate monitoring and create a custom key, then deactivate monitoring, the API key will work
- if I activate monitoring, then create an API key using the previous URL (e.g. `ms-123abc733bd058e-89.lon.meilisearch.io`), my API key will be invalid
- if I activate monitoring, then deactivate it, then create an API key using edge.meilisearch.com, my API key will also be invalid

Reply:
Based on the available info, I am inclined to leave this out of the docs. The issues described in this conversation are all connected to using the API with the "incorrect" URL. We already instruct people to use the correct URL for their application, and describing possible corner cases might make the messaging less clear.

Thanks for the help, @brunoocasali!